lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -67,7 +66,7 @@ def version
       sig { params(changes: T::Array[{ uri: String, type: Integer }]).void }
       def workspace_did_change_watched_files(changes)
         # TODO: Uncomment
-        # return unless T.must(@global_state).experimental_features
+        return unless T.must(@global_state).enabled_feature?(:tapioca_addon)
```

Comment:
Need to also remove the `TODO` comment above.
```suggestion
        return unless T.must(@global_state).enabled_feature?(:tapiocaAddon)
```